app/models/authentication.rb

```diff
@@ -14,6 +14,7 @@ def self.new(*args, &block)
   encrypt_column :service_account
 
   belongs_to :resource, :polymorphic => true
+  belongs_to :manager,  :class_name => "ExtManagementSystem"
```

Comment:
right. My forthcoming patch will get rid of this.  The only addition to `authentication` table will be just the `manager_ref` to record Tower's internal ID for the credential.